deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,186 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/* -- Fonts -- */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,500;0,700;0,900;1,100;1,200;1,300;1,700;1,900&display=swap');
+
+/* -- Global styles - Variables -- */
+:root {
```

Comment:
Wowww, ik heb :root nog niet geleerd, graag wil ik dat leren van jou als je tijd hebt of wanneer mogelijk is. Als ik jou werk kijk het is heel professioneel en  mijn werk is heel simpel.

Reply:
Door deze bootcamp komt alles een beetje op z'n plek. variabelen toepassen in de root levert erg veel voordelen op.
Als je deze toepast binnen een website dan kan je alles in één keer aanpassen bovenaan je css bestand.

Je zal wel alles van tevoren moeten plannen zodat je dan een goede basis hebt staan. Ik heb nu voor mezelf een basis CSS reset document gemaakt. Ik zal deze via teams met je delen. Dan is het alleen een kwestie van basis gegevens invullen en eventueel variabelen aanpassen en toevoegen.

Variabelen daar is heel veel over te vinden op youtube. Dit heeft mij ook erg geholpen om daarmee te kunnen werken.

Mocht er inderdaad tijd en mogelijkheid zijn dan kan ik je dit ook toelichten,